force-app/scratch-org/classes/HOT_TestData.cls

```diff
@@ -212,6 +212,7 @@ public class HOT_TestData {
             serviceResource.Name = user.Name;
             serviceResource.IsActive = true;
             serviceResource.RelatedRecordId = user.Id;
+            serviceResource.HOT_IsFreelanceInterpreter__c = true;
```

Comment:
Burde vi ha to metoder for frilans/ansatt, eller sende det inn som en parameter (sender du ikke noe blir det ansatt f.eks. ).

Reply:
Ja, men om jeg endrer på det, tryner sikkert 50% av testene. Lager du en egen sak på det?

Reply:
Det meste av funksjonaliteten er på Frilanstolk, så er mest praktisk å ha en frilanstolk, om vi kun kan ha 1. Vi kan vurdere å opprette mer test-data..
Skal vi lage en egen sak på det?